### ISSUE TITLE
Error when opening diff view

### ISSUE THREAD

**Issue by LeoTT** [process-engine/bpmn-studio#568]
#### Describe your issue

Symptoms look a lot like #290.
Error happens after opening diff-view.
After the error the content of the diagram view disappears.

Here's the error message:
```
Canvas.prototype.viewbox@http://localhost:9000/scripts/vendor-bundle.js:42641:18
Overlays.prototype._addOverlay@http://localhost:9000/scripts/vendor-bundle.js:55183:43
Overlays.prototype.add@http://localhost:9000/scripts/vendor-bundle.js:54964:5
ContextPad.prototype.getPad@http://localhost:9000/scripts/vendor-bundle.js:68731:23
ContextPad.prototype._updateAndOpen@http://localhost:9000/scripts/vendor-bundle.js:68664:15
ContextPad.prototype.open@http://localhost:9000/scripts/vendor-bundle.js:68657:5
ContextPad.prototype._init/<@http://localhost:9000/scripts/vendor-bundle.js:68542:9
invokeFunction@http://localhost:9000/scripts/vendor-bundle.js:44172:12
EventBus.prototype._invokeListener@http://localhost:9000/scripts/vendor-bundle.js:44048:21
EventBus.prototype._invokeListeners@http://localhost:9000/scripts/vendor-bundle.js:44036:21
EventBus.prototype.fire@http://localhost:9000/scripts/vendor-bundle.js:43996:21
Selection.prototype.select@http://localhost:9000/scripts/vendor-bundle.js:54525:5
PropertyPanel</PropertyPanel.prototype.setFirstElement/<@http://localhost:9000/scripts/app-bundle.js:4895:17
Reader.prototype.fromXML/<@http://localhost:9000/scripts/vendor-bundle.js:47562:7
```

#### How can others reproduce the issue?

Open diff-view in Firefox.

#### What's your setup

Firefox

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by SteffenKn** [comment]
I also get this Error when opening the XML View. Can you confirm this?

**Comment by LeoTT** [comment]
Nope, xml view works for me now (except for #572, which I just noticed today).
Some time ago it did crash though. Tested it yesterday with multiple diagrams and closed issue #290, because it seemed resolved.

If no solution comes up, consider disabling the buttons and adding a tooltip like `This feature is disabled for Firefox. Please consider using Chrome or the BPMN-Studio Application.`.

**Comment by LeoTT** [comment]
My bad, yes, now I do have the same issue.
I was on the `feature/fix_width_style_issues` branch when I just tested this. Just confirmed that xml-view works there for me, but not on current develop.

Pretty sure I encountered this yesterday on develop, so it must be some change there.
Would maybe checkout earlier commits(from the time you started the feature branch) and go from there.

**Comment by SteffenKn** [comment]
This has to do with updating the xml.

```
    this.xml = await this.getXML();
```

creates the bug for some reason.

This was accidentally removed a few days ago. Therefore #290 was fixed and #567 was created.

When the diff view gets opened the xml also gets updated.
